account_financial_report/wizard/abstract_wizard.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Lorenzo Battistini @ TAKOBI
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AbstractWizard(models.AbstractModel):
+    _name = "account_financial_report_abstract_wizard"
```

Comment:
The module name is giving errors when upgrading Odoo!

def _validate_model(model):
    if "_" in model and "." not in model and not model.startswith("x_") and model not in _VALID_MODELS:
        raise SleepyDeveloperError("`{}` seems to be a table name instead of model name".format(model))
    return model

This one blocks the upgrade of Odoo to version v17.

Reply:
Has someone else already ran into this problem?

Reply:
That error is on migration scripts or Odoo code?

Reply:
Feel free to make a PR with `_name = "account.financial.report.abstract.wizard"`

Reply:
But this is not going to help of course the migration from v14 :-)
Then it needs to be changed there also.
I logged a ticket at Odoo to exclude the abstract classes from this check...because it's not going to create a model at all.
The module works in a vanilla Odoo v17, so it's an upgrade issue.